generators/app/templates/src/redux/store.js

```diff
@@ -1,25 +1,28 @@
-import { createStore, applyMiddleware, compose, combineReducers } from 'redux';
-import thunk from 'redux-thunk';
-import { routerReducer as router, routerMiddleware } from 'react-router-redux';
-import createHistory from 'history/createBrowserHistory';
+import { connectRouter, routerMiddleware } from 'connected-react-router';
+import { createBrowserHistory } from 'history';
+import { applyMiddleware, combineReducers, compose, createStore } from 'redux';
 import { reducer as form } from 'redux-form';
+import thunk from 'redux-thunk';
 
 // TODO Add this if you need it
 // import AnalyticsMiddleware from '../services/AnalyticsService';
 import { reducer as auth } from './Auth/reducer';
 
-export const history = createHistory();
+export const history = createBrowserHistory();
 
 // Add reducers here
-const reducers = combineReducers({ auth, form, router });
+const reducers = combineReducers({ auth, form });
 
 const middlewares = [routerMiddleware(history)];
 const enhancers = [];
 
 /* ------------- Thunk Middleware ------------- */
 middlewares.push(thunk);
 
-// TODO Add this if you need it
+/* ------------- Connected React Router Middleware ------------- */
+middlewares.push(routerMiddleware(history));
```

Comment:
We are pushing this middleware but it already exists in `middlewares`.
I propose to rewrite this as:
```
const middlewares = [thunk, routerMiddleware(history)];
```
So we avoid pushing things to that list. What do you think ?

Reply:
I did not realize. good idea!